protoc-gen-twirpy/README.md

```diff
@@ -1,2 +1,22 @@
 # protoc-gen-twirpy
 twirp support for python
+
+# Installing and using plugin
+1. Make sure your [GO](https://golang.org/) environment, [Protobuf](https://github.com/protocolbuffers/protobuf) compiler is properly setup.
+2. Go to Protobuf plugin directory : `cd ../protoc-gen-twirpy`
+3. Get dependencies : `go get .`
```

Comment:
check the README in the root directory, it has much simpler install instructions

Reply:
yeah, i've modified accordingly